sharktank/tests/evaluate/baseline_perplexity_scores.json

```diff
@@ -635,5 +635,14 @@
       10.818043
     ],
     "mean_perplexity": 14.87456
+  },
+  "deepseek_v3_torch": {
+    "perplexities": [
+      512.066162,
+      8907.019531,
+      4864.950684,
+      508.603119
+    ],
+    "mean_perplexity": 3698.159874
```

Comment:
Test is currently failing:
`FAILED sharktank/tests/evaluate/perplexity_torch_test.py::PerplexityTest::test_deepseek_v3 - AssertionError: np.float64(3698.159874) != np.float64(1830.414139) within 0.5 delta (np.float64(1867.745735) difference) : Current perplexity deviates baseline by -1867.745735`